_component_design/badge.md

```diff
@@ -81,18 +155,22 @@ Use a 2px border on all sides between the number and the edge of the circle.
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### In tabs
+## Variations
+
+Considering the following guidelines when using icons and placing a badge on a tab:
+
+**With an icon**: Overlap the badge on the icon by 2px and ensure that the icon can still be seen clearly. A notification badge should not contain a **0**. If there are no notifications, do not display a badge.
```

Comment:
that the icon is still seen clearly.
(active voice)

Reply:
Done.